code/modules/fishing/aquarium/aquarium.dm

```diff
@@ -115,19 +140,36 @@
 
 /obj/structure/aquarium/examine(mob/user)
 	. = ..()
-	. += span_notice("Alt-click to [panel_open ? "close" : "open"] the control panel.")
+	. += span_notice("<b>Alt-click</b> to [panel_open ? "close" : "open"] the control and feed panel.")
+	if(panel_open && reagents.total_volume)
+		. += span_notice("You can use a plunger to empty the feed storage.")
 
-/obj/structure/aquarium/AltClick(mob/user)
+/obj/structure/aquarium/AltClick(mob/living/user)
+	. = ..()
 	if(!user.can_perform_action(src))
-		return ..()
+		return
 	panel_open = !panel_open
+	balloon_alert(user, "panel [panel_open ? "open" : "closed"]")
+	if(panel_open)
+		reagents.flags |= TRANSPARENT|REFILLABLE
+	else
+		reagents.flags &= ~(TRANSPARENT|REFILLABLE)
 	update_appearance()
 
 /obj/structure/aquarium/wrench_act(mob/living/user, obj/item/tool)
 	. = ..()
 	default_unfasten_wrench(user, tool)
 	return TOOL_ACT_TOOLTYPE_SUCCESS
 
+/obj/structure/aquarium/plunger_act(obj/item/plunger/P, mob/living/user, reinforced)
+	if(!panel_open)
+		return
+	to_chat(user, span_notice("You start plunging [name]."))
+	if(do_after(user, 3 SECONDS, target = src))
+		to_chat(user, span_notice("You finish plunging the [name]."))
```

Comment:
I don't know if these need to be to_chats rather than balloon alerts but if this is the only thing I find I'll just merge anyway because I don't care that much.